dji_robomaster_ep_driver/src/joystick_teleop_node.ros2.cpp

```diff
@@ -0,0 +1,218 @@
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <utility>
+
+#include <geometry_msgs/msg/twist_stamped.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/joy.hpp>
+
+namespace dji_robomaster_ep_driver
+{
+class ControllerMapping
```

Comment:
I think the controller mappings are the exception to separate ROS1/ROS2 nodes. It would be nice to have them defined in a single header, with the right `Twist`/`Joy` message types selected by `#ifdef`, so that they don't diverge.

Reply:
Meaning, to refactor the ROS 1 node to use these mappings as well?

Reply:
Yes

Reply:
See https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/9/commits/021e781aeed994fd2d1f22b612624b792e5816c5.